Various technical notes/transition_infos.json

```
@@ -850,11 +822,6 @@
           "area_name": "Penguin Den",
           "requires": []
         },
```

The geyser leads there

```suggestion
        },
        {
          "area_id": "0x1B8833D3",
          "area_name": "Mountain Sled Run",
          "requires": []
        },
```

---

This is one of the one-way-exits. All one-way-exits are removed from the .json file and instead handled in the entrance_rando.py file.

---

I mark this as resolved because it's a duplicate of another conversation.